### ISSUE TITLE
Field Surgeon affecting stabilized soldier

### ISSUE THREAD

**Issue by ToastedAmphibian** [long-war-2/lwotc#550]
[Field Surgeon Bug.zip](https://github.com/long-war-2/lwotc/files/3620116/Field.Surgeon.Bug.zip)

I'm guessing fixing the "downed solders evacing" bug caused some kind of trigger to be missed regarding downed soldiers not getting the bonus health from field surgeon. Unless I'm misremembering, downed soldiers should have 1 health after a mission, but the Templar is ending up with 2.

**Comment by pledbrook** [comment]
Yeah, not quite sure what the expected behaviour is here. I think Field Surgeon shouldn't apply to bleeding-out soldiers. Not sure about stabilised and/or revived one though.

**Comment by pledbrook** [comment]
> Fixes to Field Surgeon not always proccing when it should, and sometimes proccing when it shouldn't.

That's from the LW2 1.4 changelog. I suspect something was changed in the highlander to fix this and I have yet to port it to the Community Highlander.
